### ISSUE TITLE
Starting fresh install: TypeError: obj[method].bind is not a function

### ISSUE THREAD

**Issue by RothAndrew** [trailsjs/trails#99]
```
┌───────────────────────────────────────────┐
│ Your Trails Application has been created! │
│ ---                                       │
│ To start your application, run: npm start │
└───────────────────────────────────────────┘

Initialized empty Git repository in /home/ec2-user/projects/trails-app/.git/
[ec2-user@ip-172-31-56-123 trails-app]$ npm start

> trails-app@0.0.0 start /home/ec2-user/projects/trails-app
> node server.js

debug: trails event: trailpack:core:constructed
debug: trails event: trailpack:repl:constructed
debug: trails event: trailpack:router:constructed
debug: trails event: trailpack:hapi:constructed
debug: trails event: trails:start
info: Starting...
debug: trails event: trailpack:repl:validated
debug: trails event: trailpack:core:validated
debug: trails event: trailpack:router:validated
debug: trails event: trailpack:hapi:validated
debug: trails event: trailpack:all:validated
debug: trails event: trailpack:repl:configured
debug: trails event: trailpack:router:configured
debug: trails event: trailpack:hapi:configured
debug: trails event: trailpack:core:configured
debug: trails event: trailpack:all:configured
debug: trails event: trailpack:repl:initialized
debug: trails event: trailpack:router:initialized
error:
 TypeError: obj[method].bind is not a function
    at /home/ec2-user/projects/trails-app/node_modules/trailpack-core/lib/context.js:47:35
    at arrayEach (/home/ec2-user/projects/trails-app/node_modules/lodash/index.js:1289:13)
    at Function.<anonymous> (/home/ec2-user/projects/trails-app/node_modules/lodash/index.js:3345:13)
    at /home/ec2-user/projects/trails-app/node_modules/trailpack-core/lib/context.js:46:9
    at /home/ec2-user/projects/trails-app/node_modules/lodash/index.js:3395:24
    at /home/ec2-user/projects/trails-app/node_modules/lodash/index.js:3073:15
    at baseForOwn (/home/ec2-user/projects/trails-app/node_modules/lodash/index.js:2046:14)
    at Function.mapValues (/home/ec2-user/projects/trails-app/node_modules/lodash/index.js:3394:9)
    at Object.bindMethods (/home/ec2-user/projects/trails-app/node_modules/trailpack-core/lib/context.js:40:14)
    at Core.initialize (/home/ec2-user/projects/trails-app/node_modules/trailpack-core/index.js:30:40)
    at /home/ec2-user/projects/trails-app/node_modules/trails/lib/trailpack.js:51:26
    at process._tickDomainCallback (node.js:411:9)
    at Function.Module.runMain (module.js:444:11)
    at startup (node.js:136:18)
    at node.js:966:3
error:
 The application attempted to shut down, but is not in a running state. Either it is in the process of shutting down, or did not start successfully. Trails will not attempt to shut down twice.
error:
 Try increasing the loglevel to "debug" to learn more
error:
 Error: Unknown server method in string notation: FootprintController.create
    at Object.exports.contain.exports.reachTemplate.exports.assert.condition [as assert] (/home/ec2-user/projects/trails-app/node_modules/hapi/node_modules/hoek/lib/index.js:732:11)
    at Object.exports.execute.internals.prerequisites.internals.handler.internals.fromString.result.method [as fromString] (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/handler.js:206:10)
    at Object.exports.execute.internals.prerequisites.internals.handler.exports.configure (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/handler.js:131:34)
    at new module.exports.internals.Route.options (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/route.js:211:37)
    at internals.Connection._init.internals.Connection._start.internals.Connection._stop.internals.Connection._dispatch.internals.Connection.inject.internals.Connection._addRoute (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/connection.js:370:19)
    at internals.Connection._init.internals.Connection._start.internals.Connection._stop.internals.Connection._dispatch.internals.Connection.inject.internals.Connection._route (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/connection.js:362:18)
    at module.exports.internals.Plugin.internals.Plugin.register.internals.Plugin._apply (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/plugin.js:575:14)
    at module.exports.internals.Plugin.internals.Plugin.register.internals.Plugin.route (/home/ec2-user/projects/trails-app/node_modules/hapi/lib/plugin.js:545:10)
    at /home/ec2-user/projects/trails-app/node_modules/trailpack-hapi/lib/server.js:43:40
    at arrayEach (/home/ec2-user/projects/trails-app/node_modules/trailpack-hapi/node_modules/lodash/lodash.js:474:11)
    at Function.forEach (/home/ec2-user/projects/trails-app/node_modules/trailpack-hapi/node_modules/lodash/lodash.js:7633:11)
    at Object.registerRoutes (/home/ec2-user/projects/trails-app/node_modules/trailpack-hapi/lib/server.js:43:7)
    at /home/ec2-user/projects/trails-app/node_modules/trailpack-hapi/index.js:44:20
    at process._tickDomainCallback (node.js:411:9)
    at Function.Module.runMain (module.js:444:11)
    at startup (node.js:136:18)
error:
 The application attempted to shut down, but is not in a running state. Either it is in the process of shutting down, or did not start successfully. Trails will not attempt to shut down twice.
error:
 Try increasing the loglevel to "debug" to learn more
verbose: Event loop is empty. Shutting down

```

**Comment by tjwebb** [comment]
@RothAndrew try updating the versions of trails and the trailpacks to their "beta" versions. This is a result of a version mismatch that needs to be fixed in the generator.

**Comment by RothAndrew** [comment]
Thanks, that fixed it. Here's my (working) package.json. Updated all to beta except trailpack-repl. 1.0.0-alpha-3 is the latest version on npm.
```
...
"dependencies": {
    "jade": "^1.11.0",
    "lodash": "^3.10",
    "rc": "^1.1",
    "trailpack-core": "^1.0.0-beta-3",
    "trailpack-hapi": "^1.0.0-beta-2",
    "trailpack-repl": "1.0.0-alpha-3",
    "trailpack-router": "^1.0.0-beta-4",
    "trails": "latest",
    "trails-controller": "1.0.0-beta-2",
    "trails-model": "1.0.0-beta-2",
    "trails-policy": "^1.0.0",
    "trails-service": "1.0.0-beta-2",
    "winston": "^2.1.1"
  }
...
```

**Comment by tjwebb** [comment]
Awesome, thanks for the report. We're currently rolling out the beta release in stages because some of the development took several days longer than expected, and not all of the trailpacks were released at the same time.

**Comment by tjwebb** [comment]
Should be resolved in 1.0.0-beta-2: https://www.npmjs.com/package/trails
